client/shipping-label/state/selectors/errors.js

```diff
@@ -10,15 +11,19 @@ const getAddressErrors = ( { values, isNormalized, normalized, selectNormalized
 			address: __( 'This address is not recognized. Please try another.' ),
 		};
 	}
-	const { postcode, state, country } = ( isNormalized && selectNormalized ) ? normalized : values;
-	const requiredFields = [ 'name', 'address', 'city', 'postcode', 'country' ];
+	const { phone, postcode, state, country } = ( isNormalized && selectNormalized ) ? normalized : values;
+	const requiredFields = [ 'name', 'phone', 'address', 'city', 'postcode', 'country' ];
 	const errors = {};
 	requiredFields.forEach( ( field ) => {
 		if ( ! values[ field ] ) {
 			errors[ field ] = __( 'This field is required' );
 		}
 	} );
 
+	if ( ! isValidPhone( phone, country ) ) {
+		errors.phone = __( 'Invalid phone number' );
```

Comment:
What do you think about adding country to this message? In testing I tried a UK number, but it formatted as US, and then said it was invalid.